symengine/lib/symengine_wrapper.pyx

```diff
@@ -4399,16 +4365,20 @@ cdef class _Lambdify(object):
     cdef list out_shapes
     cdef readonly bint real
     cdef readonly int n_exprs
+    cdef readonly str order
     cdef vector[int] accum_out_sizes
     cdef object numpy_dtype
 
-    def __cinit__(self, args, *exprs, cppbool real=True):
+    def __cinit__(self, args, *exprs, cppbool real=True, order='C'):
         cdef vector[int] out_sizes
         self.real = real
+        self.order = order
         self.numpy_dtype = np.float64 if self.real else np.complex128
-        self.out_shapes = [get_shape(expr) for expr in exprs]
+        self.out_shapes = [np.asarray(expr).shape for expr in exprs]
```

Comment:
This is creating a numpy array just to get the shape. Isn't that expensive? I guess it doesn't matter much because this is in setup phase.

Reply:
It could be more expensive, that's true. This was mostly the get rid of some non-trivial logic (the get_shape function). Maybe we should benchmark the creation of `Lambdify` instances as well.